src/applications/static-pages/static-pages-entry.js

```diff
@@ -42,7 +45,11 @@ import {
 // Set further errors to have the appropriate source tag
 Sentry.configureScope(scope => scope.setTag('source', 'static-pages'));
 
-const store = createCommonStore(facilityReducer);
+const store = createCommonStore({
+  ...facilityReducer,
+  ...findVaFormsWidgetReducer,
+});
```

Comment:
A little nervous about messing with this `facilityReducer` here, but there wasn't a good way to add the `findVaFormsWidgetReducer`. We may want to consider ways for widgets to add their reducer dynamically.

Reply:
So I think the only thing we have to worry about here is if `facilityReducer` has any of the same key-value pairs as `findVAFormsWidgetReducer` since they will be overwritten by `findVAFormsWidgetReducer`. Of course as we add other widgets here the probability increases that we would override `facilityReducer` key-value pairs 😬

Reply:
Okay cool, then I think we're okay at the moment. I think if we see one more widget come up, then we'll want to add better support for widgets though.